### PR TITLE
SSB: Prevent possible problems from purple pills

### PR DESCRIPTION
**Pull request by ghost**
Andy will have weird stuff it it uses this move on himself

### CODE REVIEW

**Comment by KrisXV** [comment]
``` javascript
desc: "The user gains a random typing and 3 moves based on that typing (2 special moves and 1 status move). The user's attacks deal damage based off the user's Special Defense. If used again, returns the user to its original moveset and typing.",
 shortDesc: "Gains 3 random moves and typing.",
```
I'd suggest rewording these similarly to Hyperspace Fury (i.e. Adding "This move fails if the user is not Lycanium Z" at the end of the `desc` and "Lycanium Z:" at the beginning of `shortDesc`).

**Comment by ghost** [comment]
> ```js
> desc: "The user gains a random typing and 3 moves based on that typing (2 special moves and 1 status move). The user's attacks deal damage based off the user's Special Defense. If used again, returns the user to its original moveset and typing.",
>  shortDesc: "Gains 3 random moves and typing.",
> ```
> 
> I'd suggest rewording these similarly to Hyperspace Fury (i.e. Adding "This move fails if the user is not Lycanium Z" at the end of the `desc` and "Lycanium Z:" at the beginning of `shortDesc`).

we should then do that for Trivia room and Wonder Trade tbh
